kubectl-rook-ceph.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eEuo pipefail
+
+usages() {
+  cat <<EOF
+Usage: kubectl rook [commands] [args] [<flags>]
+Provide common management and troubleshooting tools for the Rook Ceph storage provider as a Krew plugin.
```

Comment:
This is description and not usage. See `man sed` or `man grep` for classic examples. Or `kubectl --help`. Kubectl does a nice thing in that it shows the usage last so that the thing most people want to know is not at the top of its long usage text, which we will end up having.

I would suggest keeping it simple and following kubectl's lead

``` 
Description:
  kubectl rook-ceph provides common management and troubleshooting tools for Rook-Ceph.
  
Commands:
  ceph - run any 'ceph' CLI command as given
    args  - any 'ceph' CLI arg (e.g., status)
    flags - any 'ceph' CLI flag (e.g., --format=json-pretty)
    
Usage:
  kubectl rook-ceph [command] [command args...] [command flags...]
```